mrrc/mrrc_logger.py

```diff
@@ -0,0 +1,57 @@
+from locale import nl_langinfo, CODESET
+from os import fdopen, dup
+import logging
+import sys
+
+MRRC_INFO_LOGGER_FORMAT='%(asctime)s : %(message)s'
```

Comment:
I think it's too complicated TBH. We should simply be calling getLogger in the class to init the logger for that specific class.

Reply:
You mean this mrrc_logger.py file is too complicated? Or just this format?  I think this format is universal control of the format for the "mrrc-uploader" logger which will be used in all other places. I don't think we will have any other loggers.